slick.grid.js

```diff
@@ -1,14 +1,11 @@
 /**
  * @license
- * (c) 2009-2013 Michael Leibman
- * michael{dot}leibman{at}gmail{dot}com
- * http://github.com/mleibman/slickgrid
+ * (c) 2015 Coatue Management LLC
+ * http://github.com/coatue/slickgrid
  *
  * Distributed under MIT license.
  * All rights reserved.
  *
- * SlickGrid v3.5

```

Comment:
I got rid of this because I didn't want to keep track of version numbers in multiple places.

At this stage, the only ones we need to track are in `bower.json` and git tags.